app/jobs/memberships/send_membership_request_mailer_job.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class SendMembershipRequestMailerJob < CaseflowJob
```

Comment:
```suggestion
class Memberships::SendMembershipRequestMailerJob < CaseflowJob
```

This will allow for initialization by specs which will make it so that the `require` there can be removed.